### DISCUSSION TITLE
Filtr shippingFromPoland dostępny w każdej kategorii

### DISCUSSION THREAD

**Discussion by olianet** [allegro/allegro-api#5333] [Suggestions - rejected]
Dzień dobry,

 aktualnie filtrowanie shippingFromPoland jest dostępne tylko dla wybranych kategorii sprzedażowych. W związku tym, że pojawia się bardo dużo zagranicznych ofert szczególnie w kategorii Zabawki i podkategoriach, bardzo pomocna byłaby możliwość filtrowania ofert tylko od polskich sprzedawców w zapytaniu _offers/listing_.

[Przykładowy wynik wyszukiwania](https://allegro.pl/kategoria/klocki-lego-17865?string=21170&order=p) obrazujący skalę problemu, pierwsze 60 ofert to oferty zagraniczne.

Z góry dziękujemy :)

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ aktualnie zasobu GET /offers/listing nie ma w publicznym API.
